docs/operations/ROLLBACK_PROCEDURE.md

```
@@ -460,15 +460,13 @@ done
 
 **Immediate** (within 5 minutes of decision):
 
-- [ ] Post in #incidents Slack channel
+- [ ] Email notification to team (jurie@phoenixvc.tech)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)